### PR TITLE
Update Slovak translation

### PR DESCRIPTION
**Pull request by KaldonM**
- fixed CrewLink to BetterCrewLink
- added new strings

### CODE REVIEW

**Comment by JKohlman** [comment]
Hey sorry, just saw this - looks like you PR'ed into nightly so I changed that to feature/openAllPlatforms as that's what this is about. I just have one quick change below if you don't mind then I'll merge it ASAP

File: static/locales/sk/translation.json [hunk 1]
```diff
@@ -128,12 +128,18 @@
...
-		"error_admin": "Nepodarilo sa pripojiť k Among Us.\n Prosím, znovu otvor CrewLink ako administrátor.",
-		"error_unsupport": "Tvoja verzia Among Us nie je CrewLinkom podporovaná."
+		"error_admin": "Nepodarilo sa pripojiť k Among Us.\n Prosím, znovu otvor BetterCrewLink ako administrátor.",
+		"error_platform": "Nezistená žiadna platforma",
+		"error_unsupport": "Tvoja verzia Among Us nie je BetterCrewLinkom podporovaná."
+	},
+	"platform": {
+		"steam": "Steam",
+		"epicgames": "Epic Games",
+		"microsoft": "Microsoft Store"
```
Comment (JKohlman) [inline static/locales/sk/translation.json:142]: Please change to just "Microsoft"

Comment (JKohlman) [inline static/locales/sk/translation.json]: Please change to just `"microsoft": "Microsoft"`

**Review by JKohlman — CHANGES_REQUESTED** [review]

File: static/locales/sk/translation.json (hunk 1 above)
Comment (JKohlman) [reply static/locales/sk/translation.json]: _Technically_ I could have done this myself in your repo but I didn't want to push to your repo without you knowing about it. If you didn't want me to have push access in your repo uncheck this first box in the righthand column and uncheck the second box when creating a PR in the future: https://imgur.com/a/2HT7FMv

**Comment by KaldonM** [comment]
Done :)

**Review by JKohlman — APPROVED** [review]

**Pull request merged**